src/test/resources/fixtures/hmh-ri/caliperEventAssessmentStarted.json

```diff
@@ -0,0 +1,58 @@
+{
+  "@context":"http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "type":"AssessmentEvent",
+  "id":"urn:uuid:uniqueIdPerEvent",
+  "actor":{
+    "id":"https://https://www.hmhco.com/ed/users/userRefId",
+    "type":"Person",
+    "name" : "Casandra Rath"
+  },
+  "action":"Started",
+  "object":{
+    "id":"https://www.hmhco.com/readinginventory/quiz/c35603f7a3434e2cb74ef14d60e70f42/instance/belonging/to/student/userRefId",
+    "type":"Assessment",
+    "name":"HMH-RI Quiz Name - Reading Inventory",
+    "dateToStartOn":"2016-11-14T05:00:00.000Z",
+    "dateToSubmit":"2016-11-18T11:59:59.000Z",
+    "maxAttempts":1,
+    "maxSubmits":1,
+    "maxScore":25.0,
+    "version":"1.0"
+  },
+  "generated":{
+    "id":"https://www.hmhco.com/readinginventory/quiz/c35603f7a3434e2cb74ef14d60e70f42/instance/belonging/to/student/userRefId/attempts/1",
+    "type":"Attempt",
+    "assignee":"https://https://www.hmhco.com/ed/users/userRefId", (same as actors id)
+    "assignable":"https://www.hmhco.com/readinginventory/quiz/c35603f7a3434e2cb74ef14d60e70f42/instance/belonging/to/student/userRefId", (same as object id)
+    "count":1,
+    "dateCreated":"2016-11-15T10:15:00.000Z",
+    "startedAtTime":"2016-11-15T10:15:00.000Z" (current date+time)
+  },
+  "eventTime":"2016-11-15T10:15:00.000Z",
+  "edApp":{
+    "id":"https://www.hmhco.com/readinginventory",
+    "type":"SoftwareApplication",
+    "version":"v1"
+  },
+  "group": {
+    "id": "https://url/to/ids/district/school/sections/sectionId1",
+    "type": "CourseSection",
```

Comment:
If we use the group as a "group", rather than as a class then we'd use a type of "Group" instead of "CourseSection", or could combine them where e.g. the group is a subOrganizationOf the class CourseSection